lib/dpp/isolation/wrapDppCreateMethod.js

```diff
@@ -0,0 +1,68 @@
+const invokeFunctionFromIsolate = require('./invokeFunctionFromIsolate');
+
+const dppErrors = [];
+
+/**
+ * Wraps dpp create method.
+ * @param {function():module:isolated-vm.Context} bootstrapIsolate -
+ * function to bootstrap an isolate
+ * @param {string} objectPath - path to a object in a global namespace withing the isolate,
+ * i.e. dpp.stateTransition
+ * @param {string} methodName - method name of the above object to call. For example, 'create'
+ * @param {Object} executionOptions - isolated-vm execution options
+ * @param {DashPlatformProtocol} dpp - external dpp
+ * to create models from jsons
+ * @param {string} modelType -
+ * can be 'document', 'stateTransition', 'identity', 'dataContract'
+ * @returns {dppCreateMethod}
+ */
+function wrapDppCreateMethod(
+  bootstrapIsolate, objectPath, methodName, executionOptions, dpp, modelType,
+) {
+  /**
+   * Invokes create method from dpp and returns an instance of corresponding dpp model
+   * @typedef dppCreateMethod
+   * return {StateTransition|Document|Identity|DataContract}
+   */
+  return async function dppCreateMethod(...args) {
```

Comment:
Please add more comments. The logic is not trivial here :)

Reply:
Done